### DISCUSSION TITLE
Jest Example does not sent anything to ES

### DISCUSSION THREAD

**Discussion by vince6881** [rfoltyns/log4j2-elasticsearch#93] [General]
Hi, i'm tryng to execute jest example by sending something to ElasticSearch 8.11.1

I got this error:
![image](https://github.com/rfoltyns/log4j2-elasticsearch/assets/147629903/aab0a1c4-dab1-4e98-b8ee-847c378fca9f)

any idea to resolve?

**Comment by vince6881** [comment]
The index template is correctly created.

**Comment by rfoltyns** [comment]
Thank you 👍 

Example templates are for ES 7.x. I'll add 8.x examples and set them as defaults this week.

**Comment by rfoltyns** [comment]
Done [8b761be](https://github.com/rfoltyns/log4j2-elasticsearch-examples/commit/8b761be862a0751fc2e39e08dc7129237401e7b9)
